charts/rancher-monitoring/v0.0.7/charts/exporter-kubelets/templates/servicemonitor.yaml

```diff
@@ -85,7 +82,18 @@ spec:
       sourceLabels:
       - pod
       targetLabel: pod_name
+    {{- else }}
+    - action: labeldrop
+      regex: (^id$|^image$|^name$|^cpu$)
```

Comment:
It seems the `labeldrop` only works in <v1.16.

Reply:
fixed